### ISSUE TITLE
"Add puzzle" dialog contents should be cleared after add/close

### ISSUE THREAD

**Issue by jpd236** [deathandmayhem/jolly-roger#2274]
When you fill out fields in the "Add puzzle" dialog and then either close it or add the puzzle, the next time you click "Add puzzle", the previous contents will still be present in the dialog.

It seems to me that the contents should be cleared. It would be relatively rare for any of the fields to carry over from puzzle to puzzle - maybe tags, but even then it's rare, and autocomplete helps a lot. Also, refreshing the page clears the old data, which makes it feel more like a bug than intended behavior.

**Comment by zarvox** [comment]
Yeah, I think the current behavior here is coincidental more so than intended.  I've felt similarly about wanting successful submission at least to clear title and URL, though I never really bothered getting around to it because when keying in e.g. the first 3-5 puzzles for a new round, it's often been convenient to keep the `group:` tag(s).  But I agree that autocomplete probably mostly covers that, and it'd feel neater/more consistent.
